src/app/components/services/util.serviec.spec.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
I intentionally left "s" out so when you write UtilService it is more succinct than UtilsService, what do you think? also since Util as in Utility is decorating Service, it make more sense to use singular, doesn't it?